src/transaction_sender/service.zig

```diff
@@ -166,19 +164,19 @@ pub const Service = struct {
         );
         defer rpc_client.deinit();
 
-        while (!self.exit.load(.unordered)) {
+        while (!self.exit.load(.monotonic)) {
             std.time.sleep(self.config.pool_process_rate.asNanos());
             if (self.transaction_pool.count() == 0) continue;
             var timer = try Timer.start();
 
             try self.processTransactions(&rpc_client);
-            self.metrics.process_transactions_latency_millis.set(timer.lap().asMillis());
+            self.metrics.process_transactions_millis.set(timer.lap().asMillis());
```

Comment:
minor implementation detail, conversion of Durations to a unit other than nanos performs a division which if we are just recording metrics probably isn't necessary

Reply:
we can do this as a follow-up, don't think it's really important for merging this one.